src/constructs/loadbalancing/alb/application-listener.test.ts

```diff
@@ -246,4 +246,34 @@ describe("The GuHttpsApplicationListener class", () => {
       ],
     });
   });
+
+  test("uses HTTPS protocol by default", () => {
```

Comment:
I think this is already covered by [`sets default props`](https://github.com/guardian/cdk/blob/8125805fe5ece34996da2e031213e30a9a43711f/src/constructs/loadbalancing/alb/application-listener.test.ts#L121-L124)?

Reply:
Correct, didn't catch this thanks.